src/components/Navbar/Menu.tsx

```diff
@@ -10,7 +10,7 @@ const Menu: React.FC<MenuProp> = ({ showMenu, hideMenu }) => {
     if (!showMenu) return;
 
     return (
-        <div className="oveflow-hidden fixed right-0 top-0 z-50 w-full max-w-[65%] border px-10 py-24 backdrop-blur-xl">
+        <div className="oveflow-hidden gradient fixed right-0 top-0 z-50 h-screen max-w-[65%] border-l border-black px-10 py-24">
```

Comment:
@teoh4770

I don't know how to feel about the menu having a gradient instead

![image](https://github.com/LaurierHawkHacks/Landing/assets/46619361/0be2961b-b17e-4b3e-91d5-52d14f27e64b)

Reply:
Well, you wanna keep the transparent background still?